deps/build.jl

```diff
@@ -5,94 +5,91 @@
 # print to stderr, since that is where Pkg prints its messages
 eprintln(x...) = println(STDERR, x...)
 
-include("ipython.jl")
-const ipython, ipyvers = find_ipython()
+juliaprofiles = Array(String,0)
 
-if ipyvers < v"1.0.0-dev"
-    error("IPython 1.0 or later is required for IJulia, got $ipyvers instead")
-else
-    eprintln("Found IPython version $ipyvers ... ok.")
-end
+@windows_only begin
+    ipython_version_to_install = "2.4.1"
```

Comment:
One question is still whether we should hardcode the IPython version that is installed or just always get the latest. Right now it is hard coded. I'm worried that if we just get the latest version instead things might break. E.g., as soon as 3.0 is released, it would be used, and who knows whether we might need to adjust things. So I would vote to keep things as they are with a hardcoded version, but would like to hear other input as well. It does mean we would have to keep this line updated.